src/main/resources/plugin.yml

```diff
@@ -4,6 +4,7 @@ author: Offz
 main: com.mineinabyss.mobzy.MobzyPlugin
 api-version: 1.16
 depend: [Geary, WorldGuard, ProtocolLib, ProtocolBurrito]
+softdepend: [ ModelEngine ]
```

Comment:
Yeah that should definitely be there, are you sure this isn't what was causing issues?

Reply:
I tried it with just this and it still showed pigs, it's because model engine has a scheduler that runs on the first tick to register stuff i think